ROADMAP.md

```diff
@@ -4,32 +4,37 @@ This document defines a high level roadmap for Karmada development and upcoming
 Community and contributor involvement is vital for successfully implementing all desired items for each release.
 We hope that the items listed below will inspire further engagement from the community to keep karmada progressing and shipping exciting and valuable features.
 
+## 2022 H1
+- Multi-cluster HA scheduling policy
+  * spread by region
+  * spread by zone
+  * spread by provider
+- Multi-cluster Ingress
+- Multi-cluster HPA (Horizontal Pod Autoscaling)
+- Federated resource quota
+- API reference
+- [Karmada website](https://karmada.io/) refactor
+- Policy-based governance, risk, and compliance
+- Multi-cluster DNS (cluster identity)
+- Global search across clusters
+- Scheduling re-balancing
 
-## 2021 Q1
-- K8s native API support for multi-cluster applications
-- Multi-cluster application status aggregation
-- Per-cluster configuration customization for applications
-- Cluster Affinity Scheduling Policy
-- Multi-cluster HA scheduling policy: spread by cluster
-- Cluster registry; Support register existing clusters
-
-## 2021 Q2
-- Multi-cluster application automatic migration (DR)
-- Dynamically split replicas of applications based on multiple dimensions
-- Taint Toleration for clusters
-- Multiple sync mode: Push, Pull
+## 2022 H2
+- Karmada Dashboard - alpha release
+- Karmada scalability baseline (performance report)
+- Cluster addons
```

Comment:
Is this about the configuration and installation of karmada related projects?  e.g. karmada dashborad, multi-cluster-ingress-nginx，or crd of other resources.

Reply:
Yes, something like that. not only the components of Karmada, including other tools from Kubernetes ecosystem, like `argocd`, `fulxcd`, etc.